obspy/core/util/base.py

```diff
@@ -246,27 +247,27 @@ def _getOrderedEntryPoints(group, subgroup=None, order_list=[]):
 
 
 ENTRY_POINTS = {
```

Comment:
Good point. I'll revert that. Now that you mentioned it my plugins are not working anymore...